src/js/modules/infragistics.ui.videoplayer.js

```diff
@@ -49,223 +49,1826 @@
+							banners: [{
+								imageUrl: "../Content/images/Banner1.png",
+								link: "http://quince.infragistics.com/",
+								times: [5, 20, 60],
+								visible: false,
+								closeBanner: true,
+								animate: true,
+								autohide: true,
+								hidedelay: 10000,
+								width: "200px",
+								height: "67px",
+								css: "bannerCss"
+							}]
+						});
+
+					//Get
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					var css = banners[0].css;
+
+					//Set
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					banners[0].css = "bannerCss";
+				```
+				*/
 				css: null
 			}],
-			/* type="object" Get or set an array of commercials objects that will be displayed when the video is playing. */
+			/* type="object" Get or set an array of commercials objects that will be displayed when the video is playing.
+			```
+				//Initialize
+				$(".selector").igVideoPlayer({
+					commercials: {
+						linkedCommercials: [
+						{
+							sources: [  "videos/QuinceIntro_1.h264.mp4",
+										"videos/QuinceIntro_1.webmvp8.webm",
+										"videos/QuinceIntro_1.theora.ogv" ],
+							startTime: 20,
+							title: "Quince Presentation p1",
+							link: "http://quince.infragistics.com/"
+						},
+						{
+							sources: [  "videos/QuinceIntro_Part3_1.h264.mp4",
+										"videos/QuinceIntro_Part3_1.webmvp8.webm",
+										"videos/QuinceIntro_Part3_1.theora.ogv" ],
+							startTime: 100,
+							title: "Quince Presentation p3",
+							link: "http://quince.infragistics.com/"
+						}]
+					}
+				});
+
+				//Get
+				var commercials = $(".selector").igVideoPlayer("option", "commercials");
+
+				//Set
+				var commercials = {
+					linkedCommercials: [
+					{
+						sources: [  "http://dl.infragistics.com/pg/2011-1/web/shared/videoplayer/videos/QuinceIntro_1.h264.mp4",
+									"http://dl.infragistics.com/pg/2011-1/web/shared/videoplayer/videos/QuinceIntro_1.webmvp8.webm",
+									"http://dl.infragistics.com/pg/2011-1/web/shared/videoplayer/videos/QuinceIntro_1.theora.ogv" ],
+						startTime: 20,
+						title: 'Quince<br/>Presentation<br/>p1',
+						link: 'http://quince.infragistics.com/'
+					}],
+					adMessage: {
+						hideDelay: 3000
+					}
+				};
+				$(".selector").igVideoPlayer("option", "commercials", commercials);
+			```
+			*/
 			commercials: {
-				/* type="array" Get or set an array of linked commercial objects. A linked commercial is a separate video file that will be played in the specified position of the original movie clip by the startTime property. This feature is useful if you have frequently changing outside commercial sources. */
+				/* type="array" Get or set an array of linked commercial objects. A linked commercial is a separate video file that will be played in the specified position of the original movie clip by the startTime property. This feature is useful if you have frequently changing outside commercial sources.
```

Comment:
startTime should be an option link.

`[startTime](ui.igvideoplayer#options:commercials.linkedCommercials.startTime)`